jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/HandlerRegistry.java

```diff
@@ -0,0 +1,56 @@
+package io.opentelemetry.contrib.jfr.metrics;
+
+import io.opentelemetry.api.metrics.MeterProvider;
+import io.opentelemetry.contrib.jfr.metrics.container.ContainerConfigurationHandler;
+import io.opentelemetry.contrib.jfr.metrics.cpu.LongLockHandler;
+import io.opentelemetry.contrib.jfr.metrics.memory.ObjectAllocationOutsideTLABHandler;
+import io.opentelemetry.contrib.jfr.metrics.network.NetworkReadHandler;
+import io.opentelemetry.contrib.jfr.metrics.network.NetworkWriteHandler;
+import io.opentelemetry.contrib.jfr.metrics.cpu.ContextSwitchRateHandler;
+import io.opentelemetry.contrib.jfr.metrics.cpu.OverallCPULoadHandler;
+import io.opentelemetry.contrib.jfr.metrics.memory.G1GarbageCollectionHandler;
+import io.opentelemetry.contrib.jfr.metrics.memory.GCHeapSummaryHandler;
+import io.opentelemetry.contrib.jfr.metrics.memory.ObjectAllocationInNewTLABHandler;
+
+
+import java.util.*;
+import java.util.stream.Stream;
+
+public class HandlerRegistry {
+  private static final String SCHEMA_URL = "https://opentelemetry.io/schemas/1.6.1";
+  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.contrib.jfr";
+  private static final String INSTRUMENTATION_VERSION = "1.7.0-SNAPSHOT";
+
+  private final List<RecordedEventHandler> mappers;
+
+  private HandlerRegistry(List<? extends RecordedEventHandler> mappers) {
+    this.mappers = new ArrayList<>(mappers);
+  }
+
+  public static HandlerRegistry createDefault(MeterProvider meterProvider) {
+    var otelMeter = meterProvider.get(INSTRUMENTATION_NAME, INSTRUMENTATION_VERSION, null);
```

Comment:
Should this be using the SCHEMA_URL from above?

Reply:
That's why I put it in there, but @jkwatson is of the opinion it should be `null` for now.